src/axom/sidre/tests/sidre_datastore_unit.cpp

```diff
@@ -277,7 +277,7 @@ TEST(sidre_datastore, create_destroy_buffers_views)
 int psrand(int min, int max)
 {
   // Returns a pseudorandom int in [min, max].  Note the closed interval.
-  int range = max - min + 1;
+  long range = max - min + 1;
   return min + int(range * rand() / (RAND_MAX + 1.0));
```

Comment:
Technically, could this cast to `int` also overflow in the same way?

Reply:
probably. Maybe I'm wrong, but I don't think the `int` cast even does anything here. Since the return type is `int` won't the cast to `int` happen implicitly.

Reply:
I meant that if `min` and `int(range)` are sufficiently high, then adding them could overflow in the same way that `max-min+1` could overflow.